tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,5 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert item not in vendor.inventory
+    assert result == False
```

Comment:
Nice work on explicitly checking these things! One thing that you could add some assert statements that checks to see if the original items are still in is the inventory as well. 
```suggestion
    assert result == False
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
    assert item not in vendor.inventory
```